gossip/snowballnetwork.go

```diff
@@ -52,7 +53,11 @@ func (snb *snowballer) Started() bool {
 	return snb.started
 }
 
-func (snb *snowballer) start(ctx context.Context, done chan error) {
+func (snb *snowballer) start(startCtx context.Context, done chan error) {
+	sp := opentracing.StartSpan("snowballer.start")
```

Comment:
Can we throw `gossip` in here somewhere so it will match my saved search in Kibana? Or we could decide on some other common thing to search on.